docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,42 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. If your device is currently flashed with the proprietary
+firmware please refer to the [Initial deployment](initial-deployment.md)
+documentation.
+
+The update process may be different, depending on the currently installed
+Dasharo firmware version.
+
+For simplicity of the update process, we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md).
```

Comment:
On the one hand, we suggest using DTS, but we also describe why. What is the purpose? I guess that for those who want a manual process, if that is the case, this is what I would write after mentioning DTS: `We recommend DTS, but if you really want to do this manually please follow guide below` or something like that.

This may be copied and pasted from another guide, so other guides should be fixed also; if you have no resources for that, feel free to issue a bug against other documentation.

Reply:
https://github.com/Dasharo/docs/pull/794/commits/d737a9610ee8cc3237969a7fe7bba2948700009b

Reply:
https://github.com/Dasharo/dasharo-issues/issues/760